Sources/Views/GiftsV2.swift

```diff
@@ -0,0 +1,128 @@
+import Dependencies
+import StyleguideV2
+import TaggedMoney
+import Transcripts
+
+public struct GiftsV2: HTML {
+  @Dependency(\.currentUser) var currentUser
+  @Dependency(\.siteRouter) var siteRouter
+
+  public init() {}
+
+  public var body: some HTML {
+    PageHeader {
+      "Gifts"
+    } blurb: {
+      """
+      Purchase a 3, 6, or 12 month subscription for a friend, colleague or loved one.
+      """
+    }
+
+    PageModule(theme: .content) {
+      LazyVGrid(columns: [.desktop: [1, 1, 1]]) {
+        Lane("3 months", annualPricePerMonth: 54) {
+          "One-time payment"
+        } features: {
+          baseFeatures
+        } callToAction: {
+          Button(color: .purple) {
+            "Choose Gift"
+          }
+          .attribute("href", siteRouter.path(for: .gifts(.plan(.threeMonths))))
+        }
+
+        Lane("6 momths", annualPricePerMonth: 108) {
+          "One-time payment"
+        } features: {
+          baseFeatures
+        } callToAction: {
+          Button(color: .purple) {
+            "Choose Gift"
+          }
+          .attribute("href", siteRouter.path(for: .gifts(.plan(.sixMonths))))
+        }
+
+        Lane("1 year", annualPricePerMonth: 168) {
+          "One-time payment"
+        } features: {
+          li { "22% off the 3 and 6 month gift options" }
+            .color(.black)
+            .backgroundColor(.yellow)
```

Comment:
Could make the highlight extend a bit and pop more:

```suggestion
            .backgroundColor(.yellow)
            .inlineStyle("margin", "-2px")
            .inlineStyle("padding", "2px")
```